image-customization.lua

```diff
@@ -195,3 +195,10 @@ end
 if target('bcm27xx') then
     packages(pkgs_hid)
 end
+
+if target('ramips', 'mt7621') or target('mediatek', 'filogic') then
```

Comment:
I think filogic is OK for now, but the other one does not seem correct. 

If you open https://openwrt.org/toh/views/toh_extended_all (don't do this on your phone!) and filter `WLAN Hardware` for `MT79` you can see that both mt7622 and mt7621 are affected. 

If you filter for subtarget mt7621, you will also notice that many many mt7621 device do not have mt7915. I think we should avoid putting this auto-reboot on devices that don't absolutely need it. This will cause more harm than give us benefits.

I think for starters, we can probably manually get the list from Gluon targets and identify which ones need the workaround. So far, only devices that were added with the newest Gluon version are affected anyway.

Reply:
can you assist here with the list?
but first we need to check why the build failed anyway (atm i am unable to download the logfile or view it)

Reply:
There is at least one device which has mt7603 for 2.4GHz and mt7915 for 5GHz ([Ubiquiti UniFi 6 Lite](https://openwrt.org/toh/hwdata/ubiquiti/ubiquiti_unifi_6_lite)).
The package does check if `/sys/kernel/debug/ieee80211/phy*/mt76/rf_regval` exists before trying to read it - so this will not do anything if the device does not have this driver.

https://github.com/ffac/gluon-packages/blob/main/ffac-mt7915-hotfix/files/lib/gluon/mt7915/reboot-on-error.sh#L2

So I think it is fine the way it is :) one might want to add mt7622 - though we did not have any device for now, but it would probably be right as well (and does not harm either as said before)

Reply:
added mt7622 as well an as the Packages checks first bevor trying i think we are fine with target approach